### PR TITLE
Filter adeptes by zone instead of referent tag

### PR DESCRIPTION
**Pull request by renanbr**

### CODE REVIEW

File: src/ManagedUsers/ManagedUsersFilter.php [hunk 1]
```diff
@@ -318,27 +317,27 @@ public function setIncludeCitizenProjectHosts(?bool $value): void
...
 
-    public function addReferentTag(ReferentTag $referentTag): void
+    public function addManagedZone(Zone $zone): void
     {
-        $this->referentTags[] = $referentTag;
+        $this->managedZones[] = $zone;
     }
 
-    public function removeReferentTag(ReferentTag $referentTag): void
+    public function removeManagedZone($zone): void
```
Comment (ottaviano) [inline src/ManagedUsers/ManagedUsersFilter.php:332]: typehint missing

File: src/Form/AbstractConnectedUserFormType.php [hunk 2]
```diff
@@ -48,4 +52,15 @@ protected function getReferentTags(): array
 
         return $user->getManagedArea()->getTags()->toArray();
     }
+
+    protected function getZones(): array
```
[the remark this answers is not in the document]
Comment (renanbr) [reply src/Form/AbstractConnectedUserFormType.php]: addressed in b284628c3cc8cd6c6099020c982e7c026825dc51

File: src/ManagedUsers/ManagedUsersFilter.php (hunk 1 above)
Comment (renanbr) [reply src/ManagedUsers/ManagedUsersFilter.php]: addressed in aecc91a7d9c2e379f23e6daa4e4f4c9e99abfce6

File: src/Form/ManagedUsers/AbstractManagedUsersFilterType.php [hunk 3]
```diff
@@ -26,7 +28,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
+                'remote_params' => ['space' => $options['space_type']],
+                'class' => Zone::class,
+                'primary_key' => 'id',
+                'minimum_input_length' => 2,
+                'page_limit' => 25,
+                'delay' => 250,
+                'language' => 'fr',
+                'placeholder' => 'Zones',
+                'autostart' => false,
+            ])
```
[the remark this answers is not in the document]
Comment (renanbr) [reply src/Form/ManagedUsers/AbstractManagedUsersFilterType.php]: addressed in 7631bc53a49c9ab3a668a2e5f309c102f239a907

**Review by ottaviano — APPROVED** [review]

**Pull request merged**